src/components/Home/Hero.tsx

```diff
@@ -62,6 +72,11 @@ export default function Hero() {
             loop
             muted
             playsInline
+            style={{
+              WebkitMaskImage: "-webkit-radial-gradient(white, black)",
```

Comment:
this is a super hacky hack to prevent having a thin black bar at the top of the video on iphones. seriously the stuff apple makes me do is just ridiculous